apps/clock/manifest.webapp

```diff
@@ -15,6 +15,12 @@
     "audio-channel-alarm":{},
     "moz-extremely-unstable-and-will-change-webcomponents":{}
   },
+  "datastores-owned": {
+    "alarms": {
+      "access" : "readwrite",
```

Comment:
@mcav, so can we have r+ on the review so the the QAs can start testing once the merge is done.

Thanks :)